android/anc/src/main/java/org/smartregister/fhircore/anc/data/family/FamilyDetailRepository.kt

```diff
@@ -32,11 +32,13 @@ import org.smartregister.fhircore.engine.util.DispatcherProvider
 class FamilyDetailRepository
 @Inject
 constructor(
-  private val familyId: String,
-  private val fhirEngine: FhirEngine,
-  private val dispatcherProvider: DispatcherProvider
+  val fhirEngine: FhirEngine,
+  val familyItemMapper: FamilyItemMapper,
+  val dispatcherProvider: DispatcherProvider
 ) {
 
+  lateinit var familyId: String
+
   fun fetchDemographics(): LiveData<Patient> {
```

Comment:
Can we remove the `LiveData` return type from the Repository

Reply:
Yea man. Let me do that